packages/mobile/src/screens/wallet-connect/components/ConfirmRemoveWalletDrawer.tsx

```diff
@@ -64,37 +43,11 @@ export const ConfirmRemoveWalletDrawer = () => {
   if (visibleState === false) return null
 
   return (
-    <NativeDrawer
+    <ConfirmationDrawer
```

Comment:
nice

Reply:
yeah im happy i was able to kill a few birds here